src/acceptance-test/java/org/zalando/nakadi/webservice/utils/ZookeeperTestUtils.java

```diff
@@ -13,6 +14,7 @@ public static CuratorFramework createCurator(final String zkUrl) {
         final CuratorFramework curator = CuratorFrameworkFactory.builder()
                 .connectString(zkUrl)
                 .retryPolicy(new RetryNTimes(5, 500))
+                .sessionTimeoutMs((int)TimeUnit.SECONDS.toMillis(10))
```

Comment:
Previous value was 40 seconds, and I was bored to wait for so much time between test runs (because old zk sessions were still present and therefore were still holding ephemeral nodes).
On production it is 6 seconds.